kubeless-ui.jsonnet

```diff
@@ -33,7 +33,9 @@ local host = "kubeless-ui";
           serviceAccountName: $.serviceAccount.metadata.name,
           containers_+: {
             default: kube.Container("ui") {
-              image: "bitnami/kubeless-ui:v0.2.3",
+              // FIXME: point to real tag
+              image: "bitnami/kubeless-ui:latest",
+              imagePullPolicy: "Always",
```

Comment:
Remove `imagePullPolicy` - the k8s default is already exactly what we want: `Always` if `:latest`, else `IfNotPresent`

Reply:
good point, done